potato_trait.obo

```diff
@@ -7496,11 +8382,10 @@ namespace: PotatoScale
 [Term]
 id: CO_330:1000003
 name: Variables
-is_a: CO_330:1000000 ! CGIAR Potato trait ontology
```

Comment:
add back the parent relationship to the ontology root term

Reply:
I removed it because the Variables class would be at the same level as the trait classes (agronomic, biotic, ...). I checked cassava before doing so, and in cassava, Variables is not under this root term